Neos.Utility.MediaTypes/Tests/Unit/MediaTypesTest.php

```diff
@@ -10,7 +10,6 @@
  * information, please view the LICENSE file which was distributed with this
  * source code.
  */
-
```

Comment:
this new line is ok. I think other files are the same way with one empty line after the header comment

Reply:
Yes, we want that line.